web_services/foxx/brightid/initdb.js

```diff
@@ -20,16 +20,20 @@ const collections = {
 };
 
 // deprecated collections should be added to this array after releasing
-// second update to allow 2 last released versions work together 
+// second update to allow 2 last released versions work together
 const deprecated = [
   'removed',
   'newGroups',
   'usersInNewGroups',
 ];
 
 const indexes = [
-  {'collection': 'verifications',  'fields': ['name']},
-  {'collection': 'verifications',  'fields': ['user']}
+  {'collection': 'verifications', 'fields': ['user'], 'type': 'persistent'},
+  {'collection': 'verifications', 'fields': ['name'], 'type': 'persistent'},
+  {'collection': 'sponsorships', 'fields': ['expireDate'], 'type': 'ttl', 'expireAfter': 0},
+  {'collection': 'sponsorships', 'fields': ['contextId'], 'type': 'persistent'},
+  {'collection': 'connections', 'fields': ['level'], 'type': 'persistent'},
+  {'collection': 'groups', 'fields': ['seed'], 'type': 'persistent'},
```

Comment:
I found one more field used by the[ `getState()` function](https://github.com/BrightID/BrightID-Node/blob/a1b6f48d318057433e040dfe9e5f7fc84bdb79aa/web_services/foxx/brightid/db.js#L760-L761) 
```suggestion
  {'collection': 'groups', 'fields': ['seed'], 'type': 'persistent'},
  {'collection': 'operations', 'fields': ['state'], 'type': 'persistent'},
```